opm/simulators/wells/VFPProdProperties.hpp

```diff
@@ -64,10 +64,10 @@ class VFPProdProperties {
                  const EvalWell& aqua,
                  const EvalWell& liquid,
                  const EvalWell& vapour,
-                 const double& thp,
-                 const double& alq,
-                 const double& explicit_wfr,
-                 const double& explicit_gfr,
+                 const Scalar& thp,
+                 const Scalar& alq,
+                 const Scalar& explicit_wfr,
+                 const Scalar& explicit_gfr,
```

Comment:
It didn't make a lot of sense before, but perhaps it's a little more useful now.  I'll nevertheless ask why we're passing these parameters as references instead of values here.

Reply:
Digging in history it seems they were made const-refs in context of old ADB code, so it should be safe to change to values, hence done.